packages/client3/components/design-system/input/input.tsx

```diff
@@ -103,60 +109,109 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
         <label htmlFor={_id}>{label}</label>
```

Comment:
The label from the figma file and here are different:

Figma:
<img width="545" alt="image" src="https://user-images.githubusercontent.com/14227409/189365303-29fbe77c-7e1b-43fa-a032-1cad52abff1a.png">

Storybook
<img width="1470" alt="image" src="https://user-images.githubusercontent.com/14227409/189365363-871604cd-ae76-4e6b-a83b-da14a45d7a31.png">

The label should be at left of the field

Reply:
I thought we're not using labels...?